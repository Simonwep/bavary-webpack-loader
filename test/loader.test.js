import {compile, execute, getCompiler, readAsset} from './helper';

describe('Loader', () => {

    it('Should work without options', async () => {
        const compiler = getCompiler('index.js');
        const stats = await compile(compiler);

        expect(execute(readAsset('main.bundle.js', compiler, stats))).toMatchSnapshot('result');
        expect(Object.keys(stats.compilation.assets)).toMatchSnapshot('assets');
        expect(stats.compilation.warnings).toMatchSnapshot('warnings');
        expect(stats.compilation.errors).toMatchSnapshot('errors');
    });
});