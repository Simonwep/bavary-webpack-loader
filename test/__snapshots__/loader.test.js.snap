// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Should work without options: assets 1`] = `
Array [
  "main.bundle.js",
]
`;

exports[`Loader Should work without options: errors 1`] = `Array []`;

exports[`Loader Should work without options: result 1`] = `"A"`;

exports[`Loader Should work without options: warnings 1`] = `Array []`;
